_includes/social-share-buttons.html

```diff
@@ -0,0 +1,12 @@
+{% assign pageurl = page.url | absolute_url %}
+{% include get_description_for_page.html %}
+
+<div class="social-share">
+  <span>Share on: </span>
+  <div class="social-share-links">
+      <a class="facebook" title="Share this on Facebook" href="https://www.facebook.com/sharer.php?u={{ pageurl }}" target="_blank" rel="noopener noreferrer"><i class="fa fa-facebook"></i></a>
```

Comment:
Do the urls not also need `url_encod`ing? We almost certainly don't want `&`s in the post url being interpreted as part of the share target url (etc.).

Reply:
Yes that's a great catch! Though we'd have to `cgi_escape` rather than `url_encode` to ensure we escape ampersands a43c52e67ad8feff48d03fa971db5db285604acb

Reply:
Hrm, it feels like this ought to have the same escaping as the text we're pushing in the urls for the template titles. The fact that this is originally a url shouldn't change the encoding scheme being used. I'm not sure which is correct, though it does feel like it ought to be consistent.

Reply:
After some playing about locally I have come up with 32f56258e77540b8f7b0093523132316f5a18dab. This ensures that ampersands still get sent through correctly and also accounts for `mailto:` not liking `+` in place of `%20`.